front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -48,7 +48,7 @@ export function ConversationTitle({
                 />
               </DropdownMenu.Button>
               <DropdownMenu.Items width={280}>
-                <div className="flex flex-col gap-y-4 py-4">
+                <div className="flex flex-col gap-y-4 px-4 py-4">
```

Comment:
NIT: can simplify with `<div className="flex flex-col gap-y-4 p-4">` here and below, which is equivalent to what you wrote. 🙏🏻